target/linux/ramips/base-files/etc/board.d/01_leds

```diff
@@ -413,6 +413,10 @@ youku-yk1)
 	set_wifi_led "$board:blue:air"
 	set_usb_led "$board:blue:usb"
 	;;
+atel-u270)
```

Comment:
A quick search revealed that the board is named ATEL ALR-U270. Please replace all occurrences of atel-u270 with alr-u270 in your PR. Please rename the dts file accordingly.